doc/source/data/doc_code/transforming_datasets.py

```diff
@@ -285,25 +285,23 @@ def repeat_dataframe(df: pd.DataFrame) -> Iterator[pd.DataFrame]:
 from typing import List
 
 # Load dataset.
-ds = ray.data.read_text("example://sms_spam_collection_subset.txt")
```

Comment:
Why couldn't this example dataset be used?

Reply:
The issue was that `read_text` now returns Datasets with Arrow format, instead of simple format, which makes it hard to show the conversions to different formats